services/web/server/tests/unit/with_dbs/conftest.py

```diff
@@ -244,23 +245,56 @@ def asyncpg_storage_system_mock(mocker):
     return mocked_method
 
 
+@pytest.fixture
+def mocked_director_subsystem(mocker):
```

Comment:
so this one is for the director-v2 right? can you name it accordingly?

Reply:
actually this is no longer used, I've removed it

Reply:
and changed the `mocked_director_api` to `mocked_director_v2_api`. this one prefaced the previous